### ISSUE TITLE
Accessibility - My Cities Add Button ( + ) Reads as "Add Button"

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Closure-App#10]
The My Cities add button should be read as "Add cities button" so a person using VoiceOver knows what they are adding. The word "Button" is not part of the text because VoiceOver knows it's a button and says so.

See screenshot below highlighting the button I'm talking about.

![closure - higlighted add button needs better text for voiceover users](https://user-images.githubusercontent.com/10423487/34884818-f11d3f36-f78b-11e7-821f-9ddccee2495a.png)

**Comment by georgepliu** [comment]
This issue has been resolved and is reflected in the current build #11

**Comment by jfc3-dol** [comment]
Thanks, for adding the text "Navigate to select cities" to the button so a screen reader (VoiceOver) user knows what the button is used for.

I'm closing this issues.
